src/Query.js

```diff
@@ -70,18 +71,18 @@ export default class Query {
         }
 
 
-        if (!mysql_handler.current_connection) {
+        if (!connection) {
             if (this.mysql[this.key].connection) {
                 this.mysql.current_connection = this.mysql[this.key].connection;
```

Comment:
Why aren't we using mysql_handler consistently when we need to use this.mysql?

Reply:
I cant remember.. that's 2 years ago

Reply:
Looks like this should be updated to mysql_handler then since they are suppose to be the same?